File: CVR/main/python/missing_number.py



def main():
    arr = [1,2,4,3]
    n=5
    missing_number = 0
    for i in range(1,6):
        missing_number = missing_number^i
    for e in arr:
        missing_number = missing_number^e
    
    print(missing_number)

main()

File: CVR/main/java/MissingNumber.java
class MissingNumber{
    public static void main(String[] args) {
        int[] arr = {1,2,3,5};
        int n = 5;
        int missingNumber = 0;

        for(int i=1;i<=n;i++){
            missingNumber = missingNumber^i;
        }

        for(int e : arr){
            missingNumber = missingNumber^e;
        }

        System.out.println(missingNumber);
    }
}